plugins/inventory/stackpath_compute.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from ansible.errors import AnsibleError
+from ansible.module_utils._text import to_native, to_text
+from ansible.plugins.inventory import (
+    BaseInventoryPlugin,
+    Constructable,
+    Cacheable
+)
+from ansible.utils.display import Display
+
+try:
+    import requests
+except ImportError:
+    raise AnsibleError(
+        'The stackpath_compute dynamic inventory plugin requires requests.'
+    )
+
+DOCUMENTATION = '''
+    name: stackpath_compute
+    plugin_type: inventory
+    short_description: StackPath Edge Computing inventory source
+    requirements:
+        - requests
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+        - Get inventory hosts from StackPath Edge Computing.
+        - Uses a YAML configuration file that ends with stackpath_compute.(yml|yaml).
+    options:
+        plugin:
+            description: >
+                A token that ensures this is a source file for the 'stackpath_compute' plugin.
+            required: True
```

Comment:
```suggestion
            required: true
            type: str
```
Please use lowercase `true`/`false` in YAML. Also, always add the correct type.